cachi2/core/package_managers/pip.py

```diff
@@ -97,51 +97,57 @@ class DistributionPackageInfo:
     index_url: str
     is_yanked: bool
 
-    pypi_checksums: set[ChecksumInfo] = field(default_factory=set)
-    user_checksums: set[ChecksumInfo] = field(default_factory=set)
+    # PyPi only returns a single checksum for a given download artifact
+    pypi_checksum: set[ChecksumInfo] = field(default_factory=set)
```

Comment:
Nit: even though PyPI seems to be consistent in providing only the `sha256` hash, I think that since we're keeping this variable as a set, we could keep its name in the plural form.

Reply:
Actually, ignore that. I remember now, from endlessly jacking around with the unit tests, that changing any of the `checksum` variables to anything other than an iterable of some kind causes huuuge problems. I'll revert the name change.